README.md

```diff
@@ -1,2 +1,18 @@
-# ACMW-Website (under development)
-Official Website for ACM-W NITK : https://acm-w-nitk.github.io/
+
+
+# ACM  W  NITK 
+<img width="180" alt="Screenshot 2019-10-06 at 12 43 03 AM" src="https://user-images.githubusercontent.com/6701828/66259727-5105e500-e7d2-11e9-9bc8-dffa5b920710.png"><br>
+Hey There! :smile:
+ Welcome to **A**ssociation for **W**omen in **C**omputing **M**achinery. ACM-W is an international organisation which supports and empowers women in all aspects of the computing field. We at ACM-W, NITK chapter wish to inspire and nurture young minds to step up and make a mark in the computing universe.
+:two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands:
+</br>
+Say Hello!  :wave:
+At ACM-W we're all ears. Feel free to drop in your feedback or suggestions, we'd love to hear from you. Leave us a mail or connect with us on our social media accounts.
+<br>
+Reach us via :email: at [acmw.nitk@gmail.com](mailto:%20acmw.nitk@gmail.com)
+
```

Comment:
Hi, there are a few more changes we would like in this :)
- [ ] Please remove emojis other than the email one. Also, to keep it more professional remove the 'Hey there'
- [ ] Ensure it is spelt as ACM-W NITK everywhere.
- [ ] The image should be the one with transparent background (images folder)
- [ ] Remove links to club officers accounts
- [ ] Add links to our other social media accounts
- [ ] The entire expansion of ACM-W should be in bold.
- [ ] Add the link to contributing guidelines.